code/game/machinery/doors/airlock.dm

```diff
@@ -324,7 +324,9 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/examine(mob/user)
 	..()
 	if(charge && !p_open && in_range(user, src))
-		user << "The maintenance panel is bulging slightly."
+		user << "<span class='warning'>The maintenance panel is slightly bulging.</span>"

```

Comment:
this isnt really your job to fix but the idea of spacemans shoving the maintenance panel on with so much force that they somehow /bend the airlock metal to make it bulge/ is p hilarious